src/server/Server.class.js

```diff
@@ -89,8 +89,7 @@ class Server {
         if (err.status === 401) {
           ctx.status = 401
           ctx.set('WWW-Authenticate', 'Basic')
-          this.app.logger.warn('UnauthorizedError: Bad Combination Login/Password')
-          ctx.body = JSON.stringify(err)
+          this.app.logger.debug('UnauthorizedError: Bad Combination Login/Password')
```

Comment:
only traced on debug? not warn?

Reply:
The warning raise questions to clients for a normal action that is explained in the frontend. Moreover the log also occurs each time the connection expires. In such a context, I don't think a warning is adapted, nor the info. Hence I passed it in debug